pkgs/applications/graphics/write_stylus/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, qt5, makeWrapper, fetchurl, makeDesktopItem }:
+stdenv.mkDerivation rec {
+  name = "write_stylus-${version}";
+  version = "209";
+
+  desktopItem = makeDesktopItem {
+    name = "Write";
+    exec = "Write";
+    comment = "a word processor for hadwriting";
+    desktopName = "Write";
+    genericName = "Write";
+    categories = "Office;Graphics";
+  };
+
+  src = fetchurl {
+    url = "http://www.styluslabs.com/write/write${version}.tar.gz";
+    sha256 = "1p6glp4vdpwl8hmhypayc4cvs3j9jfmjfhhrgqm2xkgl5bfbv2qd";
+  };
+  sourceRoot = ".";
+  unpackCmd = ''
+    tar xfz "$src"
+    '';
```

Comment:
`sourceRoot` and `unpackCmd` are probably unnecessary now that the url ends with `.tar.gz`

Reply:
couldn't build it without sourceRoot. So i just removed unpackCmd.